images/build/Dockerfiles/cli.Dockerfile

```diff
@@ -1,9 +1,10 @@
 ARG DEBIAN_FLAVOR
-FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
+FROM buildpack-deps:${DEBIAN_FLAVOR}
 ARG DEBIAN_FLAVOR
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
 
 COPY --from=buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
+COPY --from=support-files-image-for-build /tmp/oryx/ /opt/tmp
```

Comment:
I'm curious why is the `COPY` destination in `/opt/` instead of in `/tmp/`

Reply:
This is the pattern that we follow with a variety of other Dockerfiles when copying the scripts in `build` and `images` folders from the `support-files-image-for-build` intermediate image -- I guess for the CLI image we really only use it to copy over the `benv` script to `/opt/oryx`, so we can change the original copy logic, but I figured we just follow the same pattern as the other Dockerfiles here.